### PR TITLE
test(sendMessageBatch): add sqs test

### PR DESCRIPTION
**Pull request by haddasbronfman**
## Which problem is this PR solving?

no test for sqs.sendMessageBtach (aws v3)

## Short description of the changes

added test for sqs.sendMessageBatch()

## Checklist

- [ ] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts [hunk 1]
```diff
@@ -328,6 +328,60 @@ describe('instrumentation-aws-sdk-v3', () => {
...
+        expect(span.attributes[SemanticAttributes.RPC_SYSTEM]).toEqual(
+          'aws-api'
+        );
+        expect(span.attributes[SemanticAttributes.RPC_METHOD]).toEqual(
+          'SendMessageBatch'
+        );
+        expect(span.attributes[SemanticAttributes.RPC_SERVICE]).toEqual('SQS');
+        expect(span.attributes[AttributeNames.AWS_REGION]).toEqual(region);
+
+        // custom messaging attributes
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts:367]: These attributes are not custom right? The comment implies they are not in the semantic conventions. I would use a different wording like "messaging semantic attributes" or something along those lines

**Review by dyladan — APPROVED** [review]
Looks good thanks for helping increase the coverage

File: plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts (hunk 1 above)
Comment (Rauno56) [reply plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts]: Yes, that would make sense!

**Review by Rauno56 — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

**Review by blumamir — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts (hunk 1 above)
Comment (blumamir) [reply plugins/node/opentelemetry-instrumentation-aws-sdk/test/aws-sdk-v3.test.ts]: Thanks for updating the text

**Comment by blumamir** [comment]
@haddasbronfman please rebase so we can merge

**Comment by haddasbronfman** [comment]
rebase is done

**Pull request merged**